ftml/src/render/html/element/text.rs

```diff
@@ -69,8 +61,10 @@ pub fn render_code(
 
     // TODO: syntax highlighting based on 'language'
 
-    ctx.html()
-        .div()
+    ctx.html() //
+        .pre()
         .attr("class", &["code"])
```

Comment:
IIRC, the way it was "recommended" (HTML5 spec, I think?) to handle code blocks of a specific language type was to use a class, specifically `language-xxxx` or `lang-xxxx`. Should there be a default `language-none` (or something) class for unlabeled code blocks?

Reply:
I considered that but didn't end up adding one. I'll add it now.